src/metabase/models/card.clj

```diff
@@ -271,7 +271,8 @@
                 [:variable  [:template-tag (:name tag)]])
      :name    (:display-name tag)
      :slug    (:name tag)
-     :default (:default tag)}))
+     :default (:default tag)
+     :required (:required tag)}))
```

Comment:
Many tests fail with `:required nil`, so should we add this key conditionally?

Reply:
Yes, it seems it's optional, but if it's there, it should be a boolean.

Reply:
I updated it to be `(boolean (:required tag))`. Is this how you usually convert `nil` to `false`?

Reply:
Yes, that's the way to convert truthy/falsey values to boolean. The question is if this is what we need here. The other option is not to set it if `tag` doesn't have it. I can't tell just by quick look which way is the correct one.

Reply:
Given that no other key in this map is set conditionally, I am assuming it's better to leave it as is.

Reply:
But note that for example `:default` can be set to `nil`, so it wouldn't lose information.